support-frontend/assets/helpers/paymentIntegrations/readerRevenueApis.js

```diff
@@ -225,11 +222,10 @@ function postRegularPaymentRequest(
 function setPasswordGuest(
   password: string,
   guestAccountRegistrationToken: string,
-  csrf: CsrfState,
 ): Promise<boolean> {
 
   const data = { password, guestAccountRegistrationToken };
-  return logPromise(fetch(`${routes.contributionsSetPasswordGuest}`, requestOptions(data, 'same-origin', 'PUT', csrf)))
+  return logPromise(fetch(`${routes.contributionsSetPasswordGuest}`, requestOptions(data, 'same-origin', 'PUT', null)))
```

Comment:
null???!!!

Reply:
😨

Reply:
Yup!

Reply:
oh wait that's javascript, that's alright then 😆